.gitignore

```diff
@@ -62,6 +62,9 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
+# Backup Files
+*.bak
```

Comment:
why is this change needed? Looks like this `.gitignore` is messy anyways. But for change or cleanup I expect a separate PR

Reply:
when user runs the sed command above in the future, github will know to simply ignore it. I can remove this if you like.

Reply:
in that case it should also contain `sed.io`